plugins/win-dshow/win-dshow.cpp

```diff
@@ -98,9 +98,18 @@ void ffmpeg_log(void *bla, int level, const char *msg, va_list args)
 	DStr str;
 	if (level == AV_LOG_WARNING)
```

Comment:
Style:  If you change the other branches of the `if` to use curly braces, this must also use curly braces (see the [KNF style guide](https://github.com/torvalds/linux/blob/master/Documentation/process/coding-style.rst) linked from [CONTRIBUTING](https://github.com/obsproject/obs-studio/blob/master/CONTRIBUTING.rst)).